### ISSUE TITLE
encode_plus wrongly tokenizing a symbol

### ISSUE THREAD

**Issue by IbtihalFerwana** [huggingface/transformers#5091]
When I used a the bio dataset (linnaeus-IOB) to train bert on, the dataset includes the symbol (>=)  [greater than or equal], the tokenizer is separating them into two tokens 

**Expected output is either**:
`>`
`##=`
**or** 
`>= `

**Comment by LysandreJik** [comment]
What tokenizer are you using? Is it one already trained? If so, which? If not, on what did you train your tokenizer? What is your code? What is your output?

If possible, please fill the template. It's much easier to help you if you provide all the information we need.

**Comment by IbtihalFerwana** [comment]
1. Trained using a pretrained biobert for NER task:
`tokenizer = BertTokenizer.from_pretrained("monologg/biobert_v1.0_pubmed_pmc")`

2. encode_plus method
`encoded_dict = tokenizer.encode_plus(
                        sent_str,                  # Sentence to encode.
                        add_special_tokens = True, # Add '[CLS]' and '[SEP]'
                        max_length = 75,           # Pad & truncate all sentences.
                        pad_to_max_length = True,
                        return_attention_mask = True,   # Construct attn. masks.
                        return_tensors = 'pt',     # Return pytorch tensors.
                   )`
3. Dataset of 
> linnaeus-IOB: 
includes in one line `>= O`

3. Current output
the line of `>=` is split into two symbols without hashes
I get 
`> label O
= label O`

**Comment by IbtihalFerwana** [comment]
Same for the bio dataset BC4CHEMD-IOBES, it has (R), and the tokenizer split them into three tokens without hashes

**Comment by LysandreJik** [comment]
Right, I fail to understand why you think this is wrongly tokenized? This tokenizer does not have the token `>=` in its vocabulary.

You can check with:

```py
">" in tokenizer.get_vocab()  # Returns True
">=" in tokenizer.get_vocab()  # returns False
```

**Comment by IbtihalFerwana** [comment]
@LysandreJik yes sir, thank you

But I'm trying to understand why `encode_plus` did not add hashes `##` before `=`  while tokenization

I was expecting to see 
`>`
`##=`
so they would relate to the same token
but that was not the case

Is my question clear?
